src/main/kotlin/no/nav/familie/ba/mottak/task/MottaFødselshendelseTask.kt

```diff
@@ -49,6 +51,11 @@ class MottaFødselshendelseTask(private val taskRepository: TaskRepository,
         try {
             val personMedRelasjoner = personClient.hentPersonMedRelasjoner(barnetsId)
 
+            // denne kreves for at infotrygd ikke skal få fødselshendelser som ikke ligger i TPS
+            // vil feile ved forsinkelse i TPS feks i helger og helligdager.
```

Comment:
er det først når maxAntallFeil er brukt opp det produseres log-innslag, eller vil det kunne skape litt støy hvis den krever mange forsøk?

Reply:
er førts når max antall forsøk er brukt opp at vi skal logge ja,
`// lager metrikker på tasks som har feilet max antall ganger.
        if (task.status == Status.FEILET) {
            finnFeilteller(task.taskStepType).increment()
            log.error("Task ${task.id} av type ${task.taskStepType} har feilet. Sjekk familie-prosessering for detaljer")
        }`